### PR TITLE
[FIX] mrp_operations_time_control: traceback when close a MO if there  is no operation_time_line in workorder

### PR DESCRIPTION
**Pull request by agaldona**

### CODE REVIEW

**Comment by agaldona** [comment]
@esthermm @oihane @mikelarre Review please

File: mrp_operations_time_control/models/operation_time.py [hunk 1]
```diff
@@ -33,7 +33,8 @@ def _create_operation_line(self):
             'user': self.env.uid})
 
     def _write_end_date_operation_line(self):
-        self.operation_time_lines[-1].end_date = fields.Datetime.now()
+        if self.operation_time_lines:
+            self.operation_time_lines[-1].end_date = fields.Datetime.now()
```
Comment (oihane) [inline mrp_operations_time_control/models/operation_time.py:37]: Isn't it possible to use `[:-1]` or something like that?

Comment (agaldona) [inline mrp_operations_time_control/models/operation_time.py:37]: No, it raises expected singleton error.

Comment (oihane) [inline mrp_operations_time_control/models/operation_time.py:37]: And what about `[-1:]` I see you are using it in the other modified file (not sure if it has to be in this PR)

**Comment by mikelarre** [comment]
:+1:

**Comment by oihane** [comment]
Como corre prisa...

**Pull request merged**